_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,126 @@
+1. From the **Add panels** window, choose **[eCommerce] Promotion Tracking**, and then select `x` to close the panel.
+
+You've now created the following basic dashboard with a single panel, which you'll continue using throughout this tutorial.
+
+![Basic dashboard with single panel]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-basic.png)
+
+## Creating visualizations
+
+Continuing with the dashboard you created in the preceding steps, you'll create a new visualization and save it to the dashboard.
+
+1. From the dashboard toolbar, choose **Create new**, then **Visualization**.
+1. From the **New Visualization** window, choose **Vertical Bar** and then select the index pattern **opensearch_dashboards_sample_data_ecommerce**. 
+1. From the toolbar, select **Save**
+1. In the **Save visualization** window, add a title for the visualization. For example, the title for the gauge chart panel is [eCommerce] Orders.
+1.  Select **Save and return**.  
+
+The gauge chart visualization is now saved and you are taken back to the dashboard. You'll see two visualizations on the dashboard, like the following: 
+
+![Dashboard showing visualizations combined in a single view]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-combined.png)
+
+## Adding subsequent panels
+
+Continuing with the dashboard you created in the preceding steps, you'll add an existing visualization to the dashboard.
+
+1. From the dashboard toolbar, choose **Add**.
+1. From the **Add panels** window, choose **[eCommerce] Sales by Category**.
+1. Select `x` to close the **Add panels** window. 
+
+You'll see an area chart visualization display on the dashboard, as shown in the following image: 
+
+![Adding another panel to the dashboard]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-adding-panels.png)
+
+## Saving dashboards
+
+When you've finalized your dashboard, save it. If you're saving a new dashboard:
+
+1. In the toolbar, choose **Save**.
+2. On the **Save dashboard** window, enter the **Title**. The **Description** is optional.
+3. To save the time filter to the dashboard, select **Store time with dashboard**.
+4. Choose **Save**.
+
+## Customizing the look of a panel
+
+To customize the panels, you'll need to be in edit mode:
+
+1. Select **Edit** at top right in the toolbar. 
+
+Showing a legend can give readers more information, while hiding a legend can give it a cleaner look. If you want to show or hide the panel legend:
+
+1. Select the list icon in the panel's lower left corner.
+
+If you want to change the color of the panel legend:
+
+1. Select the gear icon on the area chart panel.
+2. From the **Options** window, select **Edit visualization**.
+3. From the visualization legend, select a category and then select a color from the flyout. The area chart updates with your change.
+4. Select **Save and return**. 
+
+If you want to show or hide panel title or customize it without applying the changes across the dashboard overall:
```

Comment:
"without applying the changes across the dashboard overall" isn't quite right - changing panel titles only affects the particular panel on the particular dashboard, and won't affect any other panel containing that same visualization (or any other dashboard).